classes/schema/ActionScheduler_StoreSchema.php

```diff
@@ -33,10 +36,16 @@ public function init() {
 		add_action( 'action_scheduler_before_schema_update', array( $this, 'update_schema_5_0' ), 10, 2 );
 	}
 
+	/**
+	 * Get table definition.
+	 *
+	 * @param string $table Table name.
+	 */
 	protected function get_table_definition( $table ) {
 		global $wpdb;
 		$table_name       = $wpdb->$table;
 		$charset_collate  = $wpdb->get_charset_collate();
+		// phpcs:ignore Squiz.PHP.CommentedOutCode
```

Comment:
Similar to an earlier question: is this required/what does it refer to?

```suggestion
```

Reply:
`// @see wp_get_db_schema()`

Reply:
I guess `// @see wp_get_db_schema()` on the next line, yet this rule violation is not reported when I run PHPCS (assuming that ignore statement isn't present).